modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-shipment-impl/src/main/java/com/liferay/headless/commerce/admin/shipment/internal/resource/v1_0/ShipmentResourceImpl.java

```diff
@@ -454,8 +454,9 @@ private void _updateCommerceShipment(
 			GetterUtil.get(
 				shipment.getTrackingNumber(),
 				commerceShipment.getTrackingNumber()),
-			commerceShipment.getTrackingURL(), commerceShipment.getStatus(),
-			serviceContext);
+			GetterUtil.get(
+				shipment.getTrackingURL(), commerceShipment.getTrackingURL()),
```

Comment:
I'm not sure about this, I saw this _update method is used in POST and PUT too. The PUT should always override what's already saved while the POST doesn't look so bad, but just because it shouldn't have an already save value.

Reply:
This will need a deeper refactor, it will be managed in a separate ticket, under the "tech debt" epic.